values.md

```diff
@@ -3,33 +3,50 @@
 
 ### Transparency always wins.
+
 There are countless examples of choices that we all have in our professional lives of just how honest we want to be. We choose to be completely honest, even when that is uncomfortable, or even when it would lead to short-term negative consequences. We believe that transparency is a north star, and that acting transparently is always in the long-term best interests of achieving our mission.
 
 ### We value diversity.
+
 People with different mindsets and experiences, working together, create better outcomes. This includes diversity of race and gender, as well as the diversity of academic backgrounds, socio-economic backgrounds, geographic backgrounds, ideologies, and interests.
 
-### Users are our best salespeople.
+### Users are our best advocates.
+
 We market our products and services by working with amazing humans, making them incredibly happy, and asking them to spread the word. We go above and beyond for our clients / users to invest in our future growth.
 
 ### We are humble.
+
 While we believe that we are building something exceptional, that knowledge does not make us feel proud, it makes us feel humble. It imbues us with a deep sense of responsibility towards our clients, coworkers, and the community at large.
 
-### We’re invested in the future of work.
-The employee-employer relationship of the 1900’s is a thing of the past. We fully embrace the future of work.
+### We optimize for the long-term.
 
-- We believe that employees should be able to work where and when they choose, assuming those choices are consistent with the effective performance of their responsibilities.
-- We do not believe that employees should be asked to sign overly-aggressive non-competes, IP ownership contracts, or other terms that constrict their future choices excessively.
-- We believe that compensation shouldn't create "handcuffs".
+While we pursue the mission with intense focus, we recognize that creating the change in the world that we wish to see will not happen overnight. We think, plan, and act for the long-term.
+
+### We work hard and go home.
+
+We take our responsibilities as parents, citizens, individuals, family members, and members of our various communities just as seriously as we take our commitments at work. When we are on, we are focused. When we are off, we are free to be fully disconnected.
+
+### We treat employees like people.
+
+The only way to build a team of humans who innovate in shared service to a mission is to _treat them like people_. And people only want a few things: economic security, self-determination, health, belonging, and meaning. We structure our compensation and benefits, work hours and location policies, work and management styles, and employee agreements to support this fundamental human-ness.
```

Comment:
I don't love this section -- the verbiage here is too euphemistic and hand-wavy, while also being overly prescriptive about what people _want_. Here's my take, happy to workshop or discuss!

### We are human.

We are human beings innovating in a shared service to a mission. We structure our compensation and benefits, work hours and location policies, work and management styles, and employee agreements to support this fundamental human-ness. We bring our whole selves to work, and we recognize that our identities extend beyond the work that we do.

Reply:
this is a fantastic change. I struggled writing this section; yours is much better.